src/components/header.tsx

```diff
@@ -36,6 +43,12 @@ export default function Header() {
 							disabled={ ! site.running }
 							className="[&.is-link]:text-a8c-gray-70 [&.is-link]:hover:text-a8c-blueberry !px-0 h-0 leading-4"
 							onClick={ () => getIpcApi().openSiteURL( site.id, '', { autoLogin: false } ) }
+							label={ sprintf(
```

Comment:
Should we reuse those labels for tooltip content? I've tried VoiceOver with this PR, and in those cases, when the item is active, it reads both labels one by one:

<img width="646" alt="Screenshot 2024-11-12 at 09 20 14" src="https://github.com/user-attachments/assets/74eecb26-ef71-4f6c-b2e7-29b15670e7f1">

Reply:
That's a great catch, Wojtek! Thanks.

Turns out the issue is present there with existing buttons as well, e.g. `More options` or `Close`:

![Markup on 2024-11-12 at 13:56:27](https://github.com/user-attachments/assets/893eb511-5b20-4e49-beb4-62cc728aee7a)

![Markup on 2024-11-12 at 13:56:49](https://github.com/user-attachments/assets/d3293a2c-f73c-4fb7-b5e6-0c267c1d0cfa)

The root cause is in the way the  `Button`, `DropdownMenu` and `WPModal` components (defined in `@wordpress/components`) are built: As soon as the `label` property is passed to them (which is used to trigger the Tooltip), these components add `aria-label` automatically as well. Because of that, VoiceOver reads both labels instead of just one.

I have tried a couple different solutions how to address this issue. The one that seems to be the cleanest is to rely on our custom `Tooltip` component instead. In that case we wouldn't rely on `label` prop which will prevent duplicate announcements in VoiceOver and other assistive technology apps.

I have applied the changes in the latest commit (https://github.com/Automattic/studio/pull/648/commits/333b60015ff1cc5ecbc2247df947b85a727f8ed4) so you can take a closer look.

However, further workarounds would be needed to address similar existing issue with the `DropdownMenu` and `WPModal` components that handle `More options` and `Close` buttons.

Other option could be to leave things as they are for now and accept the duplication and the way the WordPress components are built.

What do you think?

Reply:
Just noting that I have made a couple more changes to the branch in the meantime.

Reply:
Looks good, thanks for explaining the root case. I think we can leave the rest of components as is for now.